pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>org.asciidoctor</groupId>
+    <groupId>com.github.dakusui</groupId>
```

Comment:
Modified the groupId for my experiment.
This line mustn't be integrated into the main code base.

Reply:
For my manual end-to-end testing I use this module `asciidoclet` in the branch https://github.com/abelsromero/asciidoctor-maven-examples/tree/test-asciidoctlet/asciidoclet.
That does not require any publication and can use the same groupId.

1. Run `mvn install` on the asciidoclet project to get the jar(s) locally. To use the shaded JAR add the `classifier` https://github.com/abelsromero/asciidoctor-maven-examples/blob/test-asciidoctlet/asciidoclet/pom.xml#L46
2. Then, run `mvn javadoc:javadoc` in the `asciidoclet` module to get the HTML generated.

Reply:
Thanks a lot for the info! That's helpful!
As you notice, the doesn't exist in the PR anymore!